packages/web/src/components/gcds-side-nav/stories/gcds-side-nav.stories.tsx

```diff
@@ -0,0 +1,81 @@
+import { langProp } from '../../../utils/storybook/component-properties';
+
+export default {
+  title: 'Components/Side navigation',
+
+  argTypes: {
+    // Props
+    label: {
+      name: 'label',
+      control: 'text',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '-' }
+      },
+      type: {
+        required: true
+      }
+    },
+    // position: {
+    //   control: { type: 'radio' },
+    //   options: ['static', 'sticky'],
+    //   table: {
+    //     type: { summary: 'string' },
+    //     defaultValue: { summary: 'static' }
+    //   },
+    // },
+    ...langProp,
+  },
+};
+
+const Template = (args) => (`
+// Web Component (Angular, Vue)
+<gcds-side-nav
+  label="${args.label}"
+  ${args.lang != "en" ? `lang="${args.lang}"` : null}
+>
+  <gcds-nav-link href="#">Installation</gcds-nav-link>
+  <gcds-nav-link href="#">Foundations</gcds-nav-link>
+  <gcds-nav-link href="#">Components</gcds-nav-link>
+
+  <gcds-nav-group open-trigger="Contact us" menu-label="Contact">
+    <gcds-nav-link href="#">Form</gcds-nav-link>
+    <gcds-nav-link href="#">GitHub</gcds-nav-link>
+    <gcds-nav-link href="#">Slack</gcds-nav-link>
+
+    <gcds-nav-group open-trigger="Sub level 2" menu-label="sublevel">
+      <gcds-nav-link href="#">Link</gcds-nav-link>
+      <gcds-nav-link href="#">Link</gcds-nav-link>
+      <gcds-nav-link href="#">Link</gcds-nav-link>
+    </gcds-nav-group>
+  </gcds-nav-group>
+</gcds-side-nav>
+
+// React code
```

Comment:
Can we update this comment to `<!-- React code -->` so that it doesn't show in the preview?